src/Parser/RetributionPaladin/Modules/Items/WhisperOfTheNathrezim.js

```diff
@@ -23,22 +23,22 @@ class WhisperOfTheNathrezim extends Module {
 	}
 
 	on_byPlayer_damage(event) {
-		if (this.combatants.selected.hasBuff(SPELLS.WHISPER_OF_THE_NATHREZIM.id)){
-			if(event.ability.guid === SPELLS.TEMPLARS_VERDICT_DAMAGE.id || event.ability.guid === SPELLS.DIVINE_STORM.id){
-				this.damageDone += ((event.amount || 0) + (event.aborbed || 0)) * WHISPER_OF_THE_NATHREZIM_MODIFIER / (1 + WHISPER_OF_THE_NATHREZIM_MODIFIER);
+		if (this.combatants.selected.hasBuff(SPELLS.WHISPER_OF_THE_NATHREZIM_BUFF.id)){
+			if(event.ability.guid === SPELLS.TEMPLARS_VERDICT_DAMAGE.id || event.ability.guid === SPELLS.DIVINE_STORM_DAMAGE.id){
+				this.damageDone += ((event.amount || 0) + (event.absorbed || 0)) * WHISPER_OF_THE_NATHREZIM_MODIFIER / (1 + WHISPER_OF_THE_NATHREZIM_MODIFIER);
 			}
 		}
 	}
 
 	item() {
 	
-		const RetLeggoCloakUptime = this.combatants.selected.getBuffUptime(SPELLS.WHISPER_OF_THE_NATHREZIM.id) / this.owner.fightDuration;
+		const RetLeggoCloakUptime = this.combatants.selected.getBuffUptime(SPELLS.WHISPER_OF_THE_NATHREZIM_BUFF.id) / this.owner.fightDuration;
 		const fightLengthSec = this.owner.fightDuration / 1000;
 		const dps = this.damageDone / fightLengthSec;
 		return {
-			id: `spell-${SPELLS.WHISPER_OF_THE_NATHREZIM.id}`,
-			icon: <SpellIcon id={SPELLS.WHISPER_OF_THE_NATHREZIM.id} />,
-			title: <SpellLink id={SPELLS.WHISPER_OF_THE_NATHREZIM.id} />,
+			id: `spell-${SPELLS.WHISPER_OF_THE_NATHREZIM_BUFF.id}`,
+			icon: <SpellIcon id={SPELLS.WHISPER_OF_THE_NATHREZIM_BUFF.id} />,
+			title: <SpellLink id={SPELLS.WHISPER_OF_THE_NATHREZIM_BUFF.id} />,
 			result: (
 				`${formatPercentage(RetLeggoCloakUptime)}% uptime / ${formatNumber(dps)} DPS`
```

Comment:
Please use the `this.owner.formatItemDamageDone(this.damageDone)` for showing damage done.